examples/prepare_gp_optimizer.py

```diff
@@ -16,6 +16,7 @@
 from sirepo_bluesky.sirepo_ophyd import create_classes
 from sirepo_bluesky.srw_handler import SRWFileHandler
 
+import bloptools
```

Comment:
This is my suggestion to the note above:
```suggestion
import bloptools  # noqa F401
```